Song.Tests/SongWriterTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song.Tests
+{
+    public class SongWriterTests
+    {
+        [Fact]
+        public void EmptySongWriterAddSingleAnimal()
```

Comment:
It's confusing to read this, because I don't quite understand what is meant by `Empty`? Clearly you pass 1 animal.